composer.json

```diff
@@ -15,20 +15,22 @@
             "email": "torsten@appwrite.io"
         }
     ],
+"scripts": {
+        "check": "./vendor/bin/phpstan analyse --level 6 src tests",
```

Comment:
level 8 only adds 2 more errors that need to be fixed. Would you be able to change to 8 and fix those errors?